internal/txlib/add.go

```diff
@@ -47,13 +50,27 @@ func validateFileFilter(input string) error {
 	if res != 1 {
 		return errors.New("you need one <lang> in your File Filter")
 	}
+	if len(filepath.Ext(input)) < 2 {
```

Comment:
Why `<2`? We may not have a file format whose extension ends in one character yet, but if we have one in the future, we would need to patch the client.

```suggestion
	if len(filepath.Ext(input)) == 0 {
```